com.ibm.wala.cast.python.test/data/tf2_test_callbacks.py

```diff
@@ -0,0 +1,15 @@
+import tensorflow as tf
+
+# Testing API  https://www.tensorflow.org/versions/r2.9/api_docs/python/tf/distribute/MirroredStrategy#distribute_datasets_from_function
+# Making sure that function `get_dataset` is in the CG
```

Comment:
Having it in the call graph isn't enough. We need a correct summary of the function.